fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section10.java

```diff
@@ -0,0 +1,76 @@
+package ch.bbw.fabbwled.lands.book.book1;
+
+
+import ch.bbw.fabbwled.lands.book.SectionHandler;
+import ch.bbw.fabbwled.lands.book.SectionId;
+import ch.bbw.fabbwled.lands.book.SectionNode;
+import ch.bbw.fabbwled.lands.character.PlayerDto;
+import ch.bbw.fabbwled.lands.service.PlayerSession;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+
+@Component
+@RequiredArgsConstructor
+public class Section10 implements SectionHandler {
+    private final PlayerSession playerSession;
+
+    @Override
+    public SectionId getId() {
+        return new SectionId(1, 10);
+    }
+
+
+    @Override
+    public SectionNode getBody(PlayerDto current) {
+        current.addTick();
```

Comment:
care: this `addTick` will be ignored. you instead need a .`clickable(p -> p.addTick(), "tick the first empty box above"...)`
plus you have to enforce that this must be clicked once (and only once) every time we arrive here. you can for example store in `volatileSectionStore` a boolean to remember if you already clicked as long as you did not switch the section.